### DISCUSSION TITLE
What version of Ghostscript is supported with ImageMagick 6.9.10-86 version

### DISCUSSION THREAD

**Discussion by ea-vj** [ImageMagick/ImageMagick#7483] [General]
Hi Team,
We are using ghostscript version 9.27 and ImageMagick  version  6.9.10. We found that there is a vulnerability with ghostscript version 9.27 and we needed to upgrade to the latest ghostscript version. I wanted to check whether the latest version of ghostscript will be compatible with IM version 6.9.10?

**Comment by dlemstra** [comment]
It should be but feel free to come back here if it doesn't work.
